src/Scene.js

```diff
@@ -90,37 +77,22 @@ export default class TurtleGraphics extends React.Component<{}, TurtleGraphicsSt
         });
     }
 
-    clear(): void {
-        this.setState({
-            path: []
-        });
-    }
-
     render() {
-        const turtleTransform = `translate(${this.state.location.x} ${this.state.location.y}) rotate(${this.state.directionDegrees} 0 0)`;
+        // Substract 90 degrees from the character bearing as the character image is drawn upright when it is facing East
+        const robotCharacterTransform = `translate(${this.state.location.x} ${this.state.location.y}) rotate(${this.state.directionDegrees - 90} 0 0)`;
 
         return (
             <div>
                 <span
-                    className='TurtleGraphics__drawing-area'
+                    className='Scene__drawing-area'
                     role='img'
                     aria-label='Drawing area'>
                     <svg
-                        className='TurtleGraphics__svg'
+                        className='Scene__svg'
                         xmlns='http://www.w3.org/2000/svg'
-                        viewBox='-100 -100 200 200'>
-                        {this.state.path.map((pathSegment, i) => {
-                            return <line
-                                x1={pathSegment.x1}
-                                y1={pathSegment.y1}
-                                x2={pathSegment.x2}
-                                y2={pathSegment.y2}
-                                key={i} />
-                        })}
-                        <polygon
-                            className='TurtleGraphics__turtle'
-                            transform={turtleTransform}
-                            points='-6 4 6 4 0 -9'/>
+                        // Scene will have 9x5 squares with 42 unit
+                        viewBox='-189 -105 378 210'>
```

Comment:
As we discussed, let's use round numbers for the grid square size (100x100) and make the other dimensions (scene size, character size) relative to that.